packages/accordion-react/src/Accordion.tsx

```diff
@@ -1,26 +1,33 @@
 import React, { ReactNode, useState } from "react";
+// @ts-ignore
+import CoreToggle from "@nrk/core-toggle/jsx";
 
 interface Props {
     children: ReactNode;
 }
 
-export const Accordion = ({ children }: Props) => <div className="jkl-accordion">{children}</div>;
+export function Accordion({ children }: Props) {
```

Comment:
Split into two files? Then both can be Props and its maybe a bit eaier to read?

Reply:
- [x] Done!